### PR TITLE
Packaging v2 and v6.2.2

### PR DESCRIPTION
**Pull request by ericgaspar**
- app package v2

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: doc/ADMIN.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Updating WordPress
+
+As the automatic update plugin isn't working as expected, pay attention to keep your WordPress up to date from the WordPress admin panel, and not only from YunoHost admin panel. For security reason, you should control that all updates are regularly applied in WordPress admin panel as well as in YunoHost admin panel.
+
+### Security
+
+Please be aware that WordPress is known for being frequently a source of security risks (https://en.wikipedia.org/wiki/WordPress#Vulnerabilities), and also as the most popular website management system it is a target for bots and attackers.
+Some vulnerabilities might let an attacker breach into your WordPress, or even your YunoHost server (via privilege escalation).
+
+Don't forget to comply with good security principles (strong password, frequent updates, don't add unknow code in your theme/extensions…). In particular, *please keep your WordPress as up-to-date as possible*.
```
Comment (alexAubin) [inline doc/ADMIN.md]:
```suggestion
Don't forget to comply with good security principles (strong password, frequent updates, don't add unknow code in your theme/extensions…). In particular, *please keep your WordPress as up-to-date as possible*, and *do not install random, untrustworthy plug-ins*.
```

File: doc/ADMIN_fr.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### Mises à jour de WordPress
+
+Comme les mises à jour automatiques ne fonctionnent pas correctement, prenez soin de bien mettre à jour WordPress via le panneau d'administration de WordPress et pas seulement via le panneau d'administration de YunoHost. Pour des raisons de sécurité, vérifiez bien que toutes les mises à jour sont bien installées dans le panneau d'administration de WordPress comme dans le panneau d'administration de YunoHost.
+
+### Securité
+
+Soyez conscients que WordPress est connu pour avoir souvent des risques de sécurité (https://en.wikipedia.org/wiki/WordPress#Vulnerabilities), donc comme c'est le gestionnaire de sites le plus populaire il est la cible des robots et pirates.
+Des vulnérabilités peuvent offrir une brêche dans votre WordPress ou dans votre serveur YunoHost (via l'escalade des droits).
+
+N'oubliez pas d'appliquer les principes de sécurité de base (mots de passe forts, mises à jours fréquentes, ne pas ajouter du code inconnu dans le thème et les extensionts…). En particuler, *gardez votre Wordpress à jour le plus possible*.
```
Comment (alexAubin) [inline doc/ADMIN_fr.md]:
```suggestion
N'oubliez pas d'appliquer les principes de sécurité de base (mots de passe forts, mises à jours fréquentes, ne pas ajouter du code inconnu dans le thème et les extensionts…). En particuler, *gardez votre Wordpress à jour le plus possible* et *n'installez pas d'extension qui ne soit pas digne de confiance*.
```

File: scripts/install [hunk 3]
```diff
@@ -9,91 +9,28 @@
...
 fi
 
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
-ynh_script_progression --message="Storing installation settings..." --weight=2
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
-ynh_app_setting_set --app=$app --key=path --value=$path_url
 ynh_app_setting_set --app=$app --key=language --value=$language
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
```

shouldnt be needed

File: scripts/install [hunk 4]
```diff
@@ -9,91 +9,28 @@
...
-ynh_app_setting_set --app=$app --key=admin --value=$admin_wordpress
 ynh_app_setting_set --app=$app --key=multisite --value=$multisite
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
```

shouldnt be needed

**Review by alexAubin — DISMISSED** [review]
LGTM apart for minor stuff

**Review by kay0u — DISMISSED** [review]
LGMT

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Pull request merged**